app/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -40,15 +41,16 @@
         <item>墙纸</item>
         <item>@string/custom_color</item>
         <item>自定义图片…</item>
-        <item>Theme</item>
+        <item>主题背景</item>
     </string-array>
     <string-array name="background_type_values" translateable="false">
         <!-- Do not translate -->
-        <item>wallpaper</item>
-        <item>color</item>
-        <item>custom</item>
-        <item>theme</item>
+        <item>墙纸</item>
```

Comment:
Please do not translate these values. I need them in code as they are. They are only there for formatiing purposes. Thanks for translating again, though :) :+1: